src/js/elements/menu.js

```diff
@@ -1,174 +1,176 @@
-import {SubIndicator} from '../dataobjects'
-import {checkIfSubCategoryHasChildren, checkIfCategoryHasChildren, Component} from '../utils'
+import Controller from '../controller';
+import { checkIfCategoryHasChildren, checkIfSubCategoryHasChildren, Component } from '../utils';
 
 const hideondeployClsName = 'hideondeploy';
 const parentContainer = $(".data-mapper-content__list");
 const categoryTemplate = $(".data-category")[0].cloneNode(true);
 const subCategoryTemplate = $(".data-category__h2", categoryTemplate)[0].cloneNode(true);
-const indicatorTemplate = $(".data-category__h2_content", subCategoryTemplate)[0].cloneNode(true);
 const indicatorItemTemplate = $(".data-category__h4", subCategoryTemplate)[0].cloneNode(true);
 const noDataWrapperClsName = 'data-mapper-content__no-data';
 const loadingClsName = 'data-mapper-content__loading';
 
-function subindicatorsInCategory(category) {
-    let count = 0;
-    let subcategories = Object.values(category.subcategories);
-    for (const idx in subcategories) {
-        let subcategory = subcategories[idx];
-        count += subindicatorsInSubCategory(subcategory);
-    }
-
-    return count;
-}
 
 function subindicatorsInSubCategory(subcategory) {
```

Comment:
As above - this logic should live in a model

Reply:
Done.